client/py/synnax/framer/streamer.py

```diff
@@ -32,7 +32,7 @@ class _Request(Payload):
 
 class _Response(Payload):
     frame: FramePayload
-    error: ExceptionPayload | None
+    error: ExceptionPayload | str | None
```

Comment:
Is there a reason that it could be a string here?

Reply:
no, you're right, removed.